### DISCUSSION TITLE
Being able to link to Row and Automatically expand it

### DISCUSSION THREAD

**Discussion by PeterZaitsev** [grafana/grafana#38725] [General]
When I build dashboards I often want to show most relevant data above the scroll line and provide user with some way to drill down for more info if needed.  Currently I can link to URL which would open up a new page with details.  This works in some cases but is not convenient in others. 

What I would Like to be able to do as well is to link to the "Row" in the same dashboard where drilling down to more details will scroll to such "Row" and open it if it is closed.    This way I would not loose any context I have on the dashboard such as adjusting sort, only picking graph parts I want to display 

Or am I missing something it is already possible ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
